llabake/Hello-Books
# Build signin page with react redux 153971322

#### What does this PR do?
This PR implements sign in using redux to manage state
#### Description of Task to be completed?
User should be able to signin on the application
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install dependencies
* Run npm run dev to start the client server
* Run npm run start-dev to start the server-side
* Visit the port where the webpack-dev-server started, navigate to the signup page
* Fill the sign in form and click sign in,
 * If sign inis successful, a success message is displayed and the user is redirected to the home page
 * If sign in is unsuccessful, an error message is displayed.
#### What are the relevant pivotal tracker stories?
153971322
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34486753-88cb105a-efd1-11e7-8ec3-cd32adc51f6a.png)
